### PR TITLE
add Series B press release

### PR DESCRIPTION
**Pull request by sw-yx**

### CODE REVIEW

File: blog/series-b-press-release.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+slug: series-b-press-release
+title: Temporal Series B Press Release
+author: Temporal Team
+author_title: Temporal
+tags:
+  - announcement
+  - funding
+date: 2022-2-16T01:00:00Z
```
Comment (sw-yx) [inline blog/series-b-press-release.md:9]:
```suggestion
  - funding
  - press-release
date: 2022-2-16T01:00:00Z
```

File: blog/series-b-press.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+date: 2022-2-16T01:00:00Z
+---
+
+<!--truncate-->
+
+We have just announced our Series B fundraising, with a [letter from Maxim](/blog/series-b-announcement-open-letter), [announcement from Byran Offutt at Index Ventures](https://www.indexventures.com/perspectives/everything-is-a-workflow-our-investment-in-temporal/), a fun [teaser video](https://www.youtube.com/watch?v=RA7gRSXUXls) and [community Q&A](https://www.youtube.com/watch?v=E40KwlxZJFI)!
+
+Externally, we also enjoyed good press coverage. We collected them all here in one place for whoever is looking for this information in future:
+
+  - **[Geekwire](https://www.geekwire.com/2022/temporal-is-a-unicorn-developer-productivity-startup-lands-120m-at-1-5b-valuation/)**: "Temporal is riding tailwinds from the pandemic, which supercharged cloud adoption as more companies and consumers used digital services. The company said it is seeing 25-to-50% month-over-month growth in adoption. Snap, Netflix, Instacart, Qualtrics, and others are using Temporal, which now joins an elite unicorn group of Seattle companies that have eclipsed the $1 billion valuation."
```
Comment (djmagee) [inline blog/series-b-press.md:19]:
```suggestion
  - [**Geekwire**](https://www.geekwire.com/2022/temporal-is-a-unicorn-developer-productivity-startup-lands-120m-at-1-5b-valuation/)**: "Temporal is riding tailwinds from the pandemic, which supercharged cloud adoption as more companies and consumers used digital services. The company said it is seeing 25-to-50% month-over-month growth in adoption. Snap, Netflix, Instacart, Qualtrics, and others are using Temporal, which now joins an elite unicorn group of Seattle companies that have eclipsed the $1 billion valuation."
```

File: blog/series-b-press.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+tags:
+  - announcement
+  - funding
+  - press-release
+date: 2022-2-16T01:00:00Z
+---
+
+<!--truncate-->
+
+We have just announced our Series B fundraising, with a [letter from Maxim](/blog/series-b-announcement-open-letter), [announcement from Byran Offutt at Index Ventures](https://www.indexventures.com/perspectives/everything-is-a-workflow-our-investment-in-temporal/), a fun [teaser video](https://www.youtube.com/watch?v=RA7gRSXUXls) and [community Q&A](https://www.youtube.com/watch?v=E40KwlxZJFI)!
```
Comment (djmagee) [inline blog/series-b-press.md:15]:
```suggestion
We have just announced our Series B fundraising, with a [letter from Maxim](/blog/series-b-announcement-open-letter), an [announcement from Byran Offutt at Index Ventures](https://www.indexventures.com/perspectives/everything-is-a-workflow-our-investment-in-temporal/), a fun [teaser video](https://www.youtube.com/watch?v=RA7gRSXUXls), and [community Q&A](https://www.youtube.com/watch?v=E40KwlxZJFI)!
```

File: blog/series-b-press.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Externally, we also enjoyed good press coverage. We collected them all here in one place for whoever is looking for this information in future:
```
Comment (djmagee) [inline blog/series-b-press.md:17]:
```suggestion
Externally, we also enjoyed good press coverage. We collected them all here in one place for whoever is looking for this information in the future:
```

File: blog/series-b-press.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  - [**Computer Weekly**](https://www.computerweekly.com/blog/CW-Developer-Network/Temporal-calls-time-on-cloud-app-state-management): "The fact that cloud applications are getting into a (right old) state over their state management capabilities due to the inherent complexity they are now being constructed with should come as no major surprise i.e. we know that the intertask dependency management, scheduling and concurrency logic responsibilities and requirements that exist in an application’s inputs across multiple related data flows is no point-and-click fix… which is why Temporal exists."
+
+## Press Release
+
+We also shared a press release with some never-before-seen quotes from Max and our new boardmember **Mike Volpi**, and **notable users like Snap, Netflix, Descript, Datadog, and Instacart**. The full text is below.
```
Comment (djmagee) [inline blog/series-b-press.md:27]: Edit avoids using "below", per https://developers.google.com/style/word-list#letter-b.

```suggestion
We also shared the following press release, which contains never-before-seen quotes from Max and our new boardmember **Mike Volpi**, and **notable users like Snap, Netflix, Descript, Datadog, and Instacart**.
```

File: blog/series-b-press.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  - **Datadog**: leverages Temporal for database reliability and developer experience, boasting over 100 internal Temporal users. 
+  - **Snap**: is a vocal early adopter of Temporal, embracing it as a cross-organization solution for services orchestration and infrastructure deployments. “Temporal Cloud provides Snap a highly reliable and scalable foundation for Snap Stories, enabling us to deliver the amazing global experience our users have come to expect,” said Saral Jain, Director Of Engineering, Enterprise and Cloud Services at Snap Inc.
+
+
+## About Temporal
+
+The Temporal platform enables developers to create reliable applications using familiar tools that make them productive. Temporal is used by Netflix, Doordash, Snap, Box, Stripe, Hashicorp, Coinbase and others. Developers are empowered to build and operate resilient applications using developer-friendly primitives. Developers write business logic - not glue code - and Temporal takes care of the rest. More information is available at temporal.io.
```
Comment (djmagee) [inline blog/series-b-press.md:55]: Em dashes added per last para of https://developers.google.com/style/dashes#em-dashes.

```suggestion
The Temporal platform enables developers to create reliable applications using familiar tools that make them productive. Temporal is used by Netflix, Doordash, Snap, Box, Stripe, Hashicorp, Coinbase, and others. Developers are empowered to build and operate resilient applications using developer-friendly primitives. Developers write business logic—not glue code—and Temporal takes care of the rest. More information is available at temporal.io.
```

**Pull request merged**